software/spmd/hw_barrier_16x8_interrupt_test/interrupt.S

```diff
@@ -0,0 +1,107 @@
+#include "bsg_manycore_arch.h"
```

Comment:
description of test rationale; what does it intended to test, and what are the expected behaviors.